Test/LoadTasks.Tests.ps1

```diff
@@ -0,0 +1,173 @@
+
+#Requires -Version 4
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$failed = $false
+
+function GivenFile
+{
+    param(
+        $Path,
+        $Content
+    )
+
+    $Content | Set-Content -Path (Join-Path -Path $TestDrive.FullName -ChildPath $Path)
+}
+
+function Init
+{
+    $script:failed = $false
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+}
+
+function ThenError
+{
+    param(
+        $Matches
+    )
+
+    It ('should write an error') {
+        $Global:Error | Should -Match $Matches
+    }
+}
+
+function ThenFailed
+{
+    It ('should fail') {
+        $failed | Should -BeTrue
+    }
+}
+
+function ThenFile
+{
+    param(
+        $Name,
+        [Switch]
+        $Exists
+    )
+
+    It ('should load task') {
+        Join-Path -Path $TestDrive.FullName -ChildPath $Name | Should -Exist
+    }
+}
+
+function ThenTask
+{
+    param(
+        $Name,
+        [Switch]
+        $Exists
+    )
+
+    It ('should load task') {
+        Get-WhiskeyTask | Where-Object { $_.Name -eq $Name } | Should -Not -BeNullOrEmpty
+    }
+}
+
+function WhenLoading
+{
+    [CmdletBinding()]
+    param(
+    )
+
+    $Global:Error.Clear()
+    try
+    {
+        [Whiskey.Context]$context = New-WhiskeyTestContext -ForDeveloper -ConfigurationPath (Join-Path -Path $TestDrive.FullName -ChildPath 'whiskey.yml')
+        $parameter = $context.Configuration['Build'] | Where-Object { $_.ContainsKey('LoadTasks') } | ForEach-Object { $_['LoadTasks'] }
+        Invoke-WhiskeyTask -TaskContext $context -Name 'LoadTasks' -Parameter $parameter
+    }
+    catch
+    {
+        Write-Error -ErrorRecord $_
+        $script:failed = $true
+    }
+}
+
+Describe 'LoadTasks' {

```

Comment:
What happens when you run the `LoadTasks` task multiple times like when given this whiskey.yml?

```
Build:
- Pipeline:
    Name:
      - Tests
      - Package

Init:
- LoadTasks:
    Path: task.ps1

Tests:
- Pipeline:
    Name: Init
- NUnit2: ...

Package:
- Pipeline:
    Name: Init
- ProGetUniversalPackage: ...
```

Reply:
Fixed.